pennylane/wires.py

```diff
@@ -50,6 +51,9 @@ def _process(wires):
         # of considering the elements of iterables as wire labels.
         wires = [wires]
 
+    if qml.math.get_interface(wires) == "jax" and not qml.math.is_abstract(wires):
```

Comment:
Not necessary for the issue we had, but any reason we shouldn't go ahead and accept arrays from any interface? :)

Reply:
I don't see why not, although I would prefer not to implement it in this specific PR (so that we keep requirements separated)